tests/fuzz/fuzz-bolt12-tlvspan.c

```diff
@@ -0,0 +1,24 @@
+#include "config.h"
+#include <common/utils.h>
+#include <wire/wire.h>
+#include <tests/fuzz/libfuzz.h>
+
+/* Include bolt12.c directly, to gain access to tlv_span(). */
+#include "../../common/bolt12.c"
```

Comment:
`tlv_span` is public.  We shouldn't need to include `bolt12.c` file directly.

Reply:
If we add bolt12.c to the Makefile, we shouldn't need to do any stubs either.

If we're concerned about the fuzz target sizes increasing, we can start linking dependencies only to the targets that need them (and which you're already doing for this target).